requirements.txt

```diff
@@ -2,7 +2,8 @@ jupyter==1.0.0
 ipython>=7.16.3
 matplotlib==3.2.2
 seaborn==0.11.0
-pytorch==1.10.2
+# not installing pytorch from here
```

Comment:
there shouldn't be any commented code in codebase (history is maintained in commit history)